### PR TITLE
qubesos-fwupd-core

### PR DESCRIPTION
**Pull request by Asiderr**
Signed-off-by: Norbert Kamiński <norbert.kaminski@3mdeb.com>

### CODE REVIEW

File: blog/content/post/2020-07-06-qubesos-fwupd-core.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+[![asciicast](https://asciinema.org/a/v7ZiSG3Xp9fauzwFkMNZgmHgt.svg)](https://asciinema.org/a/v7ZiSG3Xp9fauzwFkMNZgmHgt)
+
+### qubes-fwupdmgr downgrade
+
+The downgrade process is similar to updating. A user chooses the device number
+and the firmware version, that will be installed on the downgraded device. Then
+the tool downloads, validates, and installs firmware downgrade `.cab` archive.
+The firmware installation proceeds with `--allow-older` flag.
+
+The downgrade process is presented in the following video.
```
Comment (macpijan) [inline blog/content/post/2020-07-06-qubesos-fwupd-core.md:158]: @Asiderr Below we have some image, not video

File: blog/content/post/2020-07-06-qubesos-fwupd-core.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+Following videos show the testing process on Qubes OS and Ubuntu.
+
+[![asciicast](https://asciinema.org/a/TgHOkLnD2YICxB0U80PVcQGqX.svg)](https://asciinema.org/a/TgHOkLnD2YICxB0U80PVcQGqX)
+
+[![asciicast](https://asciinema.org/a/pafnoJp50uQj0qKESCghS4FYW.svg)](https://asciinema.org/a/pafnoJp50uQj0qKESCghS4FYW)
+
+## Development roadmap
+
+The next step of the development process will be replacing the static paths of
```
Comment (macpijan) [inline blog/content/post/2020-07-06-qubesos-fwupd-core.md:182]: @Asiderr I would add a little bit more description of this issue with paths to binaries being changes across fwupd releases. Current sentence may not reflect what is the real issue and how we plan to solve it.

Comment (Asiderr) [reply blog/content/post/2020-07-06-qubesos-fwupd-core.md]: @macpijan I've added more information on this topic and changed the rest of this paragraph.

File: blog/content/post/2020-07-06-qubesos-fwupd-core.md (hunk 1 above)
Comment (Asiderr) [reply blog/content/post/2020-07-06-qubesos-fwupd-core.md:160]: @macpijan Yeah, there should be a video. Fixed in the last commit.

**Comment by Asiderr** [comment]
@marmarek This is the blogpost I've mentioned.

**Pull request merged**